gulpfile.babel.js

```diff
@@ -401,25 +420,27 @@ gulp.task('demos', ['demoresources'], () => {
 
   const tasks = getComponentFolders().map(component => {
     return gulp.src([
-        path.join('src', component, 'snippets', '*.html'),
-        path.join('src', component, 'demo.html')
-      ])
-      .pipe($.concat('/demo.html'))
-      // Add basic front matter.
-      .pipe($.header('---\nlayout: demo\nbodyclass: demo\ninclude_prefix: ../../\n---\n\n'))
-      .pipe($.frontMatter({
-        property: 'page',
-        remove: true
-      }))
-      .pipe($.marked())
-      .pipe((() => {
-        return through.obj((file, enc, cb) => {
-          file.page.component = component;
-          cb(null, file);
-        });
-      })())
-      .pipe(applyTemplate())
-      .pipe(gulp.dest(path.join('dist', 'components', component)));
+      path.join('src', component, 'snippets', '*.html'),
+      path.join('src', component, 'demo.html')
+    ])
+    .pipe($.concat('/demo.html'))
+    // Add basic front matter.
+    .pipe($.header(
+        '---\nlayout: demo\nbodyclass: demo\n' +
+        'include_prefix: ../../\n---\n\n'))
```

Comment:
Maybe separate PR, but we could consider using template literals to avoid awkward `\n` and such.